### DISCUSSION TITLE
0.15.0-beta3 ffmpeg oom issue

### DISCUSSION THREAD

**Discussion by flyize** [blakeblackshear/frigate#15626] [Ask A Question]
I've been having a strange issue with Frigate since 0.13 where ffmpeg tries to grab like 100GB of RAM and barfs. In 0.14, it seemed that the ffmpeg process would restart. Now it seems like that isn't happening and my camera never comes back. Is that possible?

**Comment by hawkeye217** [comment]
ffmpeg was upgraded to 7.x in 0.15. We'd need to see your config and logs to be able to help.

**Comment by flyize** [comment]
I don't expect you to troubleshoot the ffmpeg crash, as its been happening on every release since 0.13. If you're really interested in trying, I'll absolutely upload them. I'm just hoping to get the process to restart (assuming that's really the issue).

**Reply by hawkeye217** [reply]
Well, if it was working in 0.14, you can revert to ffmpeg 5.x (which was what 0.14 used) by setting your `path` to `"5.0"`, like:

```
ffmpeg:
  path: "5.0"
```

**Reply by flyize** [reply]
> Well, if it was working in 0.14, you can revert to ffmpeg 5.x (which was what 0.14 used) by setting your `path` to `"5.0"`, like:
> 
> ```
> ffmpeg:
>   path: "5.0"
> ```

Its been crashing since 0.13. So I assume its just something weird on my end. I'm okay with it, as long as ffmpeg restarts. It doesn't seem to be doing that any longer.

**Reply by NickM-27** [reply]
You'll want to make sure you're on the latest dev image. From there creating a support issue would be good as it definitely should restart the process.

**Reply by flyize** [reply]
> You'll want to make sure you're on the latest dev image. From there creating a support issue would be good as it definitely should restart the process.

At the risk of looking really dumb, what do I need to pull that? 'dev' did not seem to work.

**Reply by Vladd1980** [reply]
> Ну, если это работало в версии 0.14, вы можете вернуться к ffmpeg 5.x (который использовался в версии 0.14), установив значение `path`, `"5.0"`например:
> 
> ```
> ffmpeg:
>   path: "5.0"
> ```

can I use ffmpeg from 0.13 in the same way?

**Reply by NickM-27** [reply]
Not for rockchip, but 5.0 is the same version that was in 0.13 for the mainline builds

**Reply by Vladd1980** [reply]
> Не для Rockchip, но 5.0 — это та же версия, что была в 0.13 для основных сборок

OK. I'll formulate the question differently. Can I use the ffmpeg version for rockchip 0.13 in 0.15? If yes, what should I prescribe? ffmpeg 0.15 is completely unstable and causes problems due to which I cannot use 0.15. I wrote to you about this in another topic

**Comment by hawkeye217** [comment]
You can find builds in the packages section: https://github.com/blakeblackshear/frigate/pkgs/container/frigate

**Reply by flyize** [reply]
Is there not just a dev tag or something? Or do I just have to pull the latest?

docker pull ghcr.io/blakeblackshear/frigate:70fc539-rk

Someday I'll totally understand github...

**Reply by NickM-27** [reply]
You need to pull the latest, you want to use the build without any -suffix

**Reply by flyize** [reply]
Got it. I'll wait for it to crash. Safe to assume you'd like a new thread with logs?

**Reply by NickM-27** [reply]
yep

**Comment by flyize** [comment]
Since moving to the dev build, ffmpeg has restarted every time it crashed. I briefly tried beta4 and it did not restart ffmpeg after a crash. Suggestions?

**Reply by NickM-27** [reply]
sounds like a coincidence, nothing has changed there

**Reply by flyize** [reply]
Not sure I follow. dev=beta4?

I ran dev for at least 48 hours and ffmpeg restarted just fine. With beta4, my cam was dead in <24 hours.

**Reply by NickM-27** [reply]
There is only 1 commit between dev and beta 4, and it had nothing to do with ffmpeg

**Reply by flyize** [reply]
Okay. The original suggest was to move to dev and post logs when it happened. I assume I should just submit logs from beta4 then?

**Reply by NickM-27** [reply]
Sure
